### ISSUE TITLE
Getting `Infinity%` first time running `npx vitest` with no test files

### ISSUE THREAD

**Issue by HassanZahirnia** [vitest-dev/vitest#226]
### Describe the bug

I just followed the docs steps:
1. Run `npm install -D vitest`
2. Then I added these to my `package.json`
```json
"scripts": {
        "test": "vitest",
        "coverage": "vitest --coverage"
    },
```
3. And I ran `npx vitest`, and the output for the `Time` was :
```
No test files found

Test Files  no tests
     Tests  no tests
      Time  3.95s (in thread 0ms, Infinity%)


 PASS  Waiting for file changes...
```
There was a `Infinity%` which seemed odd to me so I thought I should report it.

### Reproduction

I believe this requires no reproduction as it's just 3 simple steps followed from vitest [guide](https://vitest.dev/guide/).

### System Info

```shell
System:
    OS: Linux 4.19 Ubuntu 20.04.3 LTS (Focal Fossa)
    CPU: (8) x64 Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz
    Memory: 2.78 GB / 6.79 GB
    Container: Yes
    Shell: 5.0.17 - /bin/bash
  Binaries:
    Node: 16.13.1 - /usr/bin/node
    Yarn: 1.22.17 - /usr/bin/yarn
    npm: 8.1.2 - /usr/bin/npm
  Browsers:
    Chrome: 96.0.4664.93
  npmPackages:
    @vitejs/plugin-vue: ^2.0.1 => 2.0.1 
    vite: ^2.7.0-beta.10 => 2.7.4 
    vitest: ^0.0.100 => 0.0.100
```

### Used Package Manager

npm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/vitest-dev/vitest/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/vitest-dev/vitest/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://vitest.dev/guide).
- [X] Check that there isn't [already an issue](https://github.com/vitest-dev/vitest/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vitest-dev/vitest/discussions) or join our [Discord Chat Server](https://chat.vitest.dev).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by nieyuyao** [comment]
Is this issue been fixed? I still reproduced on `0.9.3`.

**Comment by OmgImAlexis** [comment]
Still able to reproduce on `0.17.0`.

```
❯ npm run test uwebsocket
npm WARN config global `--global`, `--local` are deprecated. Use `--location=global` instead.

> mothership@0.0.91 test
> vitest run "uwebsocket"


 RUN  v0.17.0 C:/Users/xo/code/unraid/mothership

 ↓ src/__test__/apps/uwebsocket.test.ts (2) [skipped]

Test Files  1 skipped (1)
     Tests  2 skipped (2)
      Time  1.34s (in thread 0ms, Infinity%)
```
